slides/k8s/pv-pvc-sc.md

```diff
@@ -320,4 +320,4 @@ kubectl get pv,pvc
 :EN:- Storage provisioning
 :EN:- PV, PVC, StorageClass
 :FR:- Création de volumes
-:FR:- PV, PVC, et StorageClass
\ No newline at end of file
+:FR:- PV, PVC, et StorageClass
```

Comment:
🤔 not intended (I used the github UI to edit the doc and raise a PR automatically).

```suggestion
```